### ISSUE TITLE
Underline Class ignored for colformat_md

### ISSUE THREAD

**Issue by rexdouglass** [atusy/ftExtra#65]
The example code using the underline class fails when used alone but not when used in conjunction with another modifier like shading.color. As a work around I'm using text.color='black' 

```
data.frame(
  a = c("**bold**", "*italic*"),
  b = c("^superscript^", "~subscript~"),
  c = c("`code`", "[underline]{.underline}"),
  d = c("*[**~ft~^Extra^**](https://ftextra.atusy.net/) is*",
        "[Cool]{.underline shading.color='skyblue'}"),
  stringsAsFactors = FALSE
) %>%
  as_flextable() %>%
  colformat_md()
```

![image](https://user-images.githubusercontent.com/3146793/139288758-94c338c8-7f75-46fb-827f-91c157f9945e.png)

![image](https://user-images.githubusercontent.com/3146793/139289178-3723124f-d505-4610-a9af-610d4828d5cc.png)

**Comment by atusy** [comment]
Hi @rexdouglass 

Thank you for using ftExtra and reporting the issue, and sorry that I have missed it.
I will see what is happening during the holiday, and see if I can fix it!

**Comment by atusy** [comment]
hmmm... my code below does not reproduce your issue on R 4.1 with Ubuntu 20.04...
Would you provide a reproducible example?

```r
library(ftExtra)
tibble::tibble(x = '[foo]{.underline}') |> 
  as_flextable() |>
  colformat_md()
```

**Comment by atusy** [comment]
I close this issue because I cannot reproduce, but feel free to re-open if necessary.

**Comment by atusy** [comment]
@rexdouglass This might be related to Pandoc version, and should be already solved by v0.4.0 on CRAN.

https://github.com/atusy/ftExtra/pull/74
